Source/Xamarin/Prism.DryIoc.Forms/PrismApplication.cs

```diff
@@ -128,7 +128,7 @@ protected override void ConfigureViewModelLocator()
         /// <param name="view"><see cref="Page" /> navigated to</param>
         /// <param name="type"><see cref="Type" /> to resolve</param>
         /// <param name="navigationService">Overriding instance of <see cref="INavigationService" /></param>
-        protected virtual void ResolveTypeForPage(Page view, Type type, INavigationService navigationService)
+        protected internal virtual void ResolveTypeForPage(Page view, Type type, INavigationService navigationService)
```

Comment:
Perfect. Feel so stupid I didn't think of that before. Will implement the changes and adjust the tests. 

Thanks